src/zarr/__init__.py

```diff
@@ -1,54 +1,41 @@
-# flake8: noqa
-from zarr.codecs import *
-from zarr.convenience import (
-    consolidate_metadata,
-    copy,
-    copy_all,
-    copy_store,
-    load,
-    open,
-    open_consolidated,
-    save,
-    save_array,
-    save_group,
-    tree,
-)
-from zarr.core import Array
-from zarr.creation import (
-    array,
-    create,
-    empty,
-    empty_like,
-    full,
-    full_like,
-    ones,
-    ones_like,
-    open_array,
-    open_like,
-    zeros,
-    zeros_like,
-)
-from zarr.errors import CopyError, MetadataError
-from zarr.hierarchy import Group, group, open_group
-from zarr.n5 import N5Store, N5FSStore
-from zarr.storage import (
-    ABSStore,
-    DBMStore,
-    DictStore,
-    DirectoryStore,
-    KVStore,
-    LMDBStore,
-    LRUStoreCache,
-    MemoryStore,
-    MongoDBStore,
-    NestedDirectoryStore,
-    RedisStore,
-    SQLiteStore,
-    TempStore,
-    ZipStore,
+from __future__ import annotations
```

Comment:
this is showing up as a line-edit diff but no edits were made. I first moved the original `__init__.py` -> `v2/__init__.py`, then made a commit. Then the `v3/__init__.py` file was moved here in a second commit.